tests/tutorials/test_tutorials.py

```diff
@@ -189,6 +189,4 @@ def test_vision_cnn_visualization():
 
 def test_control_flow():
     assert _test_tutorial_nb('control_flow/ControlFlowTutorial')
-
-def test_subgraphapi():
-    assert _test_tutorial_nb('c++/subgraphAPI')
```

Comment:
Is there an issue that's tracking this? The subgraph API is something fairly new and users might be interested using it. Thus, we should make sure that our tutorials are being validated. Sorry if the tutorial is not testable - I didn't have a look.

Reply:
It is a C++ tutorial that is not testable in Jupyter